packages/react-ui/components/Kebab/Kebab.tsx

```diff
@@ -182,6 +202,7 @@ export class Kebab extends React.Component<KebabProps, KebabState> {
         if (this.state.opened) {
           this.props.onOpen();
         } else {
+          this.props.onBlur?.();
```

Comment:
При работе с клавиатуры может прийти аргумент `restoreFocus = true`. В таких случаях не надо вызывать `onBlur()`, т.к. по факту фокус сразу восстанавливается после закрытия всплывашки.
```suggestion
          !restoreFocus && this.props.onBlur?.();
```